modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/launcher/TerminatorFactory.java

```diff
@@ -0,0 +1,27 @@
+package org.ballerinalang.composer.service.workspace.launcher;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Launcher Terminator factory.
+ */
+public class TerminatorFactory {
+
+    private static final Logger logger = LoggerFactory.getLogger(TerminatorFactory.class);
+
+    public Terminator getTerminator(String os, Command command) {
+
+        if (os.equalsIgnoreCase("unix")) {
+
+            return new TerminatorUnix(command);
+
+        } else if (os.equalsIgnoreCase("windows")) {
+            return new TerminatorWindows(command);
+
+        } else {
+            logger.error("Unknown Operating System");
+        }
+        return null;
```

Comment:
since there is already an else block, lets move return null to the same for clarity